content/docs/tasks/security/authn-policy/index.md

```diff
@@ -364,16 +376,16 @@ $ kubectl exec $(kubectl get pod -l app=sleep -n legacy -o jsonpath={.items..met
 200
 {{< /text >}}
 
-## Having both namespace-level and service-level policies
+### Policy precedence
 
-Assuming we already added the namespace-level policy that enables mutual TLS for all services in namespace `foo` and observe that request from `sleep.legacy` to `httpbin.foo` are failing (see above). Now add another policy that disables mutual TLS (peers section is empty) specifically for the `httpbin` service:
+To illustrate a service-specific policy will be chosen over namespace-wide policy, let's add a policy to disable mutual TLS for `httpbin.foo` as below. Note that we already of namespace-wide policy that enables mutual TLS for all services in namespace `foo` and observe that request from `sleep.legacy` to `httpbin.foo` are failing (see above).
```

Comment:
To illustrate how a service-specific policy takes precedence over a namespace-wide policy, you can add a policy to disable mutual TLS for `httpbin.foo` as shown below. Note that you've already created a namespace-wide policy that

request -> requests